runtime/test-runtime/src/xcm_config.rs

```diff
@@ -80,17 +84,22 @@ impl WeightTrader for DummyWeightTrader {
 	}
 }
 
+type OriginConverter = (
+	pallet_xcm::XcmPassthrough<super::RuntimeOrigin>,
+	SignedAccountId32AsNative<ThisNetwork, super::RuntimeOrigin>,
```

Comment:
@bkchr Is there any reason why this has to be included? This additional converter is making 2 other integration tests fail.